Makefile

```diff
@@ -52,24 +85,24 @@ RSSPPLIB_SOURCES=$(sort $(wildcard rss/*.cpp))
 RSSPPLIB_OBJS=$(patsubst rss/%.cpp,rss/%.o,$(RSSPPLIB_SOURCES))
 RSSPPLIB_OUTPUT=librsspp.a
 
-CARGO_FLAGS+=--verbose
+CARGO_BUILD_FLAGS+=--verbose
 ifeq ($(PROFILE),1)
 ifdef CARGO_BUILD_TARGET
-NEWSBOATLIB_OUTPUT=target/$(CARGO_BUILD_TARGET)/debug/libnewsboat.a
-LDFLAGS+=-L./target/$(CARGO_BUILD_TARGET)/debug
+NEWSBOATLIB_OUTPUT=$(CARGO_TARGET_DIR)/$(CARGO_BUILD_TARGET)/debug/libnewsboat.a
+LDFLAGS+=-L$(CARGO_TARGET_DIR)/$(CARGO_BUILD_TARGET)/debug
 else
-NEWSBOATLIB_OUTPUT=target/debug/libnewsboat.a
-LDFLAGS+=-L./target/debug
+NEWSBOATLIB_OUTPUT=$(CARGO_TARGET_DIR)/debug/libnewsboat.a
+LDFLAGS+=-L$(CARGO_TARGET_DIR)/debug
 endif
 else
 ifdef CARGO_BUILD_TARGET
-NEWSBOATLIB_OUTPUT=target/$(CARGO_BUILD_TARGET)/release/libnewsboat.a
-LDFLAGS+=-L./target/$(CARGO_BUILD_TARGET)/release
+NEWSBOATLIB_OUTPUT=$(CARGO_TARGET_DIR)/$(CARGO_BUILD_TARGET)/release/libnewsboat.a
+LDFLAGS+=-L$(CARGO_TARGET_DIR)/$(CARGO_BUILD_TARGET)/release
 else
-NEWSBOATLIB_OUTPUT=target/release/libnewsboat.a
-LDFLAGS+=-L./target/release
+NEWSBOATLIB_OUTPUT=$(CARGO_TARGET_DIR)/release/libnewsboat.a
+LDFLAGS+=-L$(CARGO_TARGET_DIR)/release
 endif
-CARGO_FLAGS+=--release
+CARGO_BUILD_FLAGS+=--release
 endif
```

Comment:
It might be nice to reduce duplication here by changing the "nested if" into:
```
ifeq ($(PROFILE),1)
BUILD_TYPE=debug
else
BUILD_TYPE=release
CARGO_BUILD_FLAGS+=--release
endif

ifdef CARGO_BUILD_TARGET
NEWSBOATLIB_OUTPUT=$(CARGO_TARGET_DIR)/$(CARGO_BUILD_TARGET)/$(BUILD_TYPE)/libnewsboat.a
LDFLAGS+=-L$(CARGO_TARGET_DIR)/$(CARGO_BUILD_TARGET)/$(BUILD_TYPE)
else
NEWSBOATLIB_OUTPUT=$(CARGO_TARGET_DIR)/$(BUILD_TYPE)/libnewsboat.a
LDFLAGS+=-L$(CARGO_TARGET_DIR)/$(BUILD_TYPE)
endif
```

Warning: I did not test it.

Reply:
Indeed! It does work locally, so I pushed c35f166 to see if it works across the CI builds. Thank you for the suggestion!